modules/standard/IO.chpl

```diff
@@ -6229,6 +6229,13 @@ proc channel.writef(fmtStr:string): bool throws {
    Read arguments according to a format string. See
    :ref:`about-io-formatted-io`.
 
+   .. warning::
+
+      Intents for all arguments except the format string are `ref`. If `readf`
+      is used with formats that require an additional argument such as `%*i` and
+      `%*S`, then those arguments cannot be constants. Instead, define variables
+      and pass them.
```

Comment:
"Instead, define variables and pass them" -> "Instead, store the value into a variable and pass that." ?